source/educators/references/resources_for_course_teams.rst

```diff
@@ -67,29 +57,25 @@ a PDF version, select **v: latest** at the lower right of the page, then select
 **PDF**.
 
 .. note:: If you use the Safari browser, be aware that it does not support the
- search feature for the HTML versions of the edX guides. This is a known
+ search feature for the HTML versions of the Open edX guides. This is a known
  limitation.
 
-======
+*****
 Email
```

Comment:
Remove this section, it is entirely edx.org specific

Reply:
done!